### PR TITLE
translate service_settings.py content

### PR DESCRIPTION
**Pull request by sastels**
fixes https://github.com/cds-snc/notification-api/issues/801

Old
image

New
image

### CODE REVIEW

**Comment by anikbrazeau** [comment]
msgid "This service name is already in use"
**msgstr "Ce nom de service a dèjà été utilisé."**

msgid "Thanks for your request to go live. We’ll get back to you within one working day."
**msgstr "Merci pour votre demande d'activation du service. Nous communiquerons avec vous dans un délai d'un jour ouvrable."**

**msgid "This will suspend the service and revoke all API keys. Are you sure you want to suspend this service?"**
msgstr "Ceci suspendra le service et révoquera toutes les clés API associées. Êtes-vous certain de vouloir suspendre ce service?"

**msgid "This will resume the service. New API keys are required for this service to use the API."**
msgstr "Ceci vous permettra de reprendre le service. Vous devrez créer de nouvelles clés API pour que ce service puisse utiliser l'API."

msgid "Sending domain updated"
msgstr "Domaine d'envoi mis à jour"

msgid "Are you sure you want to delete this contact block?"
msgstr "Êtes-vous certain de vouloir supprimer ce bloc de contacts?"

msgid "Are you sure you want to delete this text message sender?"
msgstr "Êtes-vous certain de vouloir supprimer l'expéditeur de ce message texte?"

msgid " ‘SERVICE_NAME’ was deleted"
msgstr " ‘SERVICE_NAME’ a été supprimé"

File: Makefile [hunk 1]
```diff
@@ -21,6 +21,12 @@ generate-version-file: ## Generates the app version file
 test:
 	./scripts/run_tests.sh
 
+.PHONY: babel
+babel: 
+	pybabel extract -F babel.cfg -k _l -o messages.pot .
+	pybabel update -i messages.pot -d app/translations
+	pybabel compile -d app/translations
+
```
Comment (sastels) [inline Makefile:29]: Added this because I kept messing up copying and pasting out of the README :/

File: app/main/views/service_settings.py [hunk 2]
```diff
@@ -333,13 +333,13 @@ def archive_service(service_id):
         service_api_client.archive_service(service_id)
         session.pop('service_id', None)
         flash(
-            '‘{}’ was deleted'.format(current_service.name),
+            _("‘%(service_name)s’ was deleted", service_name=current_service.name),
```
Comment (smcmurtry) [inline app/main/views/service_settings.py:336]: Very cool! I didn't know you could do string interpolation in a translated string.

Comment (sastels) [reply app/main/views/service_settings.py:336]: Took me a while to notice / figure out that you need the “s” after the variable to say to treat it as a string 😅

**Pull request merged**